.github/workflows/test.yml

```diff
@@ -7,7 +7,7 @@ env:
   MEMGRAPH_PORT: 7687
   NEO4J_PORT: 7688
   NEO4J_CONTAINER: "neo4j_test"
-  OFFICIAL: "true"
+  OFFICIAL: "false"
```

Comment:
why?

Reply:
As v2.13 is not released yet, we’re running the checks on the current master (i.e. unofficial). After merging, this should be fixed in the "prepare mage for release" PR.